[2021-12-01 16:46:42,160] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: FrequencyCount.map3 manual__2021-12-01T16:45:33.951777+00:00 [queued]>
[2021-12-01 16:46:42,171] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: FrequencyCount.map3 manual__2021-12-01T16:45:33.951777+00:00 [queued]>
[2021-12-01 16:46:42,171] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-01 16:46:42,171] {taskinstance.py:1242} INFO - Starting attempt 1 of 2
[2021-12-01 16:46:42,171] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-01 16:46:42,182] {taskinstance.py:1262} INFO - Executing <Task(MapOperator): map3> on 2021-12-01 16:45:33.951777+00:00
[2021-12-01 16:46:42,188] {standard_task_runner.py:52} INFO - Started process 147 to run task
[2021-12-01 16:46:42,191] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'FrequencyCount', 'map3', 'manual__2021-12-01T16:45:33.951777+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/mapred.py', '--cfg-path', '/tmp/tmp5psnubag', '--error-file', '/tmp/tmpdjgxlbrf']
[2021-12-01 16:46:42,191] {standard_task_runner.py:77} INFO - Job 59: Subtask map3
[2021-12-01 16:46:42,227] {logging_mixin.py:109} INFO - Running <TaskInstance: FrequencyCount.map3 manual__2021-12-01T16:45:33.951777+00:00 [running]> on host 60ba21408f08
[2021-12-01 16:46:42,272] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=FrequencyCount
AIRFLOW_CTX_TASK_ID=map3
AIRFLOW_CTX_EXECUTION_DATE=2021-12-01T16:45:33.951777+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-01T16:45:33.951777+00:00
[2021-12-01 16:46:42,500] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=FrequencyCount, task_id=map3, execution_date=20211201T164533, start_date=20211201T164642, end_date=20211201T164642
[2021-12-01 16:46:42,523] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-01 16:46:42,574] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
